cmd/acr/purge.go

```diff
@@ -143,9 +147,13 @@ func newPurgeCmd(out io.Writer, rootParams *rootParameters) *cobra.Command {
 				}
 			}
 			// After all repos have been purged the summary is printed.
-			fmt.Printf("\nNumber of deleted tags: %d\n", deletedTagsCount)
-			fmt.Printf("Number of deleted manifests: %d\n", deletedManifestsCount)
-
+			if purgeParams.dryRun {
+				fmt.Printf("\nNumber of tags to be deleted: %d\n", deletedTagsCount)
+				fmt.Printf("Number of manifests to be deleted: %d\n", deletedManifestsCount)
+			} else {
+				fmt.Printf("\nNumber of deleted tags: %d\n", deletedTagsCount)
+				fmt.Printf("Number of deleted manifests: %d\n", deletedManifestsCount)
+			}
```

Comment:
Should we also update `dryRunPurge` around [line 520](https://github.com/Azure/acr-cli/blob/e425ad1399591f68e38ac69bae1e3f3599285bf6/cmd/acr/purge.go#L520)?

Reply:
changed.